recipe/meta.yaml

```
@@ -25,44 +24,44 @@ requirements:
     - python
     - cymem 2.0.6
     - cython 0.29.32
-    - cython-blis 0.7.7
     - murmurhash 1.0.7
     - numpy {{ numpy }}
     - pathy 0.10.1
     - pip
     - preshed 3.0.6
     - setuptools
-    - thinc 8.0.15
+    - thinc 8.1.10
     - wheel
   run:
     - python
     # Spacy libraries
-    - spacy-legacy >=3.0.9,<3.1.0
+    - spacy-legacy >=3.0.11,<3.1.0
     - murmurhash >=0.28.0,<1.1.0
     - cymem >=2.0.2,<2.1.0
     - preshed >=3.0.2,<3.1.0
-    - thinc >=8.0.14,<8.1.0
-    - cython-blis >=0.4.0,<0.8.0
-    - wasabi >=0.9.1,<1.1.0
+    - thinc >=8.1.8,<8.2.0
+    - wasabi >=0.9.1,<1.2.0
     - srsly >=2.4.3,<3.0.0
     - catalogue >=2.0.6,<2.1.0
     - spacy-loggers >=1.0.0,<2.0.0
-    # Exclude typer with unpinned click
-    - typer >=0.3.0,<0.5.0
+    # Exclude typer with unpinned click (>=0.3.2)
+    - typer >=0.3.2,<0.5.0
```

```suggestion
    - typer >=0.3.0,<0.5.0
```
https://github.com/explosion/spaCy/blob/v3.5.3/setup.cfg#L55

(All our typer versions have upper bounds set on click.)

---

Thanks. I updated to `typer >=0.3.0,<0.8.0`. But I wouldn't change the typer's upper bound because the maintainers always double-check if any changes can break something, see https://github.com/explosion/spaCy/pull/11720#issue-1429261529